File: 01_programming_basics/02_exercises/02_exercise_conditional_statements/07_shopping.py
budget = float(input())
number_video_cards = int(input())
number_processors = int(input())
number_ram = int(input())
video_cards_price = number_video_cards * 250
processor_price = video_cards_price * 0.35 * number_processors
ram_price = video_cards_price * 0.10 * number_ram
total = video_cards_price + processor_price + ram_price
if number_video_cards > number_processors:
    total = total * 0.85
difference = abs(budget - total)
if budget >= total:
    print(f"You have {difference:.2f} leva left!")
else:
    print(f"Not enough money! You need {difference:.2f} leva more!")
    

File: 05_JavaScriptBasics/02_Exercises/02_ConditionalStatements/07_shopping.js
function shopping(input){
    let budget = Number(input[0])
    let numVideoCards = Number(input[1])
    let numProcessors = Number(input[2])
    let numRamMemory = Number(input[3])

    let priceVideoCards = numVideoCards * 250
    let priceProcessors = numProcessors * (priceVideoCards * 0.35)
    let priceRamMemories = numRamMemory * (priceVideoCards * 0.10)
    let totalPrice = priceVideoCards + priceProcessors + priceRamMemories

    if (numVideoCards > numProcessors){
        totalPrice *= 0.85
    }
    let difference = Math.abs(totalPrice -  budget)

    if (budget >= totalPrice){
        console.log(`You have ${difference.toFixed(2)} leva left!`)
    } else {
        console.log(`Not enough money! You need ${difference.toFixed(2)} leva more!`)
    }
}

shopping(["900", "2", "1", "3"])
shopping(["920.45", "3", "1", "1"])
